parser/prism/Helpers.h

```diff
@@ -185,11 +185,9 @@ template <typename PrismNode> PrismNode *down_cast(pm_node_t *anyNode) {
     static_assert(std::is_same_v<decltype(PrismNode::base), pm_node_t>,
                   "The `down_cast` function should only be called on Prism node pointers.");
     ENFORCE(PM_NODE_TYPE_P(anyNode, PrismNodeTypeHelper<PrismNode>::TypeID),
-        "Failed to cast a Prism AST Node. Expected {} (#{}), but got {} (#{}).",
-            pm_node_type_to_str(PrismNodeTypeHelper<PrismNode>::TypeID),
-            PrismNodeTypeHelper<PrismNode>::TypeID,
-            pm_node_type_to_str(PM_NODE_TYPE(anyNode)),
-            PM_NODE_TYPE(anyNode));
+            "Failed to cast a Prism AST Node. Expected {} (#{}), but got {} (#{}).",
+            pm_node_type_to_str(PrismNodeTypeHelper<PrismNode>::TypeID), PrismNodeTypeHelper<PrismNode>::TypeID,
+            pm_node_type_to_str(PM_NODE_TYPE(anyNode)), PM_NODE_TYPE(anyNode));
```

Comment:
I feel like there are a lot of formatting changes in recent PRs -- what's up with that?